src/domain/event/eventInfo/EventsHierarchy.tsx

```diff
@@ -0,0 +1,119 @@
+import { IconAngleDown, IconAngleUp, IconLayers } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import InfoWithIcon from '../../../common/components/infoWithIcon/InfoWithIcon';
+import linkStyles from '../../../common/components/link/link.module.scss';
+import SkeletonLoader from '../../../common/components/skeletonLoader/SkeletonLoader';
+import LoadingSpinner from '../../../common/components/spinner/LoadingSpinner';
+import { getEventTypeByEventTypeId } from '../EventUtils';
+import { useSubEvents, useSubEventsQueryVariables } from '../queryUtils';
+import { EventFields, SuperEventResponse } from '../types';
+import { Eventlist } from './eventList/EventList';
+import styles from './eventList/eventList.module.scss';
+
+const EVENTS_LIST_LIMIT = 3;
+export const subEventsListTestId = 'sub-events-list';
+export const superEventTestId = 'super-event';
+
+const SubEvents: React.FC<{ event: EventFields }> = ({ event }) => {
+  const { t } = useTranslation();
+  const [isListOpen, setIsListOpen] = React.useState(false);
+  const eventType = event.typeId
+    ? getEventTypeByEventTypeId(event.typeId)
+    : 'event';
+
+  const { superEventId, variables } = useSubEventsQueryVariables(event);
```

Comment:
hmm could be moved inside useSubEvents instead of calling it here and passing many params?

Reply:
hmm never mind, it is used in 2 different places.